mmv1/third_party/terraform/services/dataproc/resource_dataproc_session_template_test.go

```diff
@@ -0,0 +1,198 @@
+package dataproc_test
+
+import (
+	"testing"
+
+	"github.com/hashicorp/terraform-plugin-testing/helper/resource"
+	"github.com/hashicorp/terraform-plugin-testing/plancheck"
+
+	"github.com/hashicorp/terraform-provider-google/google/acctest"
+	"github.com/hashicorp/terraform-provider-google/google/envvar"
+)
+
+func TestAccDataprocSessionTemplate_dataprocSessionTemplatesJupyterFullExample_update(t *testing.T) {
```

Comment:
```suggestion
func TestAccDataprocSessionTemplate_update(t *testing.T) {
```
mentioned in the earlier comment, but I'd rename these to be less confusing with the generated examples

Reply:
Ack, updated.